Add Gambit MIMIC Simulator integration (#2568)
DataDog/integrations-extras: gambit_communications_inc_gambit_mimic_simulator/README.md

# Gambit MIMIC Simulator

## Overview

Gambit MIMIC is a dynamic, real-time, high-performance SNMP, Netflow, sFlow Simulator. All Netflow and sFlow exports are recorded by the simulator and replayed with flexible customization toward Datadog agent(s) and presented with all existing available metrics of Datadog, whether built-in or Datadog marketplace integrated.

## Setup

## MIMIC Installation Instructions

### [Overview]()

MIMIC is distributed as a compressed file for each platform, either on CD-ROM or downloadable from the Web. This file needs to be uncompressed and its contents extracted on your system.

Follow the below 3 steps to install.

> After installation, please review the [MIMIC Frequently Asked Questions page][1].

Reviewer: Move this to the end

**NOTE: you only need to download MIMIC when a new release is issued.**\
The evaluation and purchased versions are identical except for the license keys. If you have already downloaded MIMIC, determine the version you are running (eg. in MIMICview `Help->About Mimic...` or by looking at the contents of the `config/version` file), and download again only if the distribution listed below is newer.

#### Step 1: Get the MIMIC distribution

If you have a CD-ROM, copy the distribution file for your platform from the CD-ROM to a temporary directory. Go to Step 2.

To download from the Web from a URL that Gambit has provided, save the distribution files for your desired platform to a temporary directory (use right mouse-button click on the link for most web browsers).

> **NOTE: make sure the entire file was downloaded and verify size.**
> 
> **NOTE: there are scam versions of any software floating around the Internet with all sorts of malware. Download only authoritative versions of MIMIC from a URL that Gambit has given you directly.**

Reviewer: ```suggestion
**NOTES:**
- Ensure the entire file was downloaded and verify the file size
- Only download verified software from Gambit
```

#### Step 2: Uncompress and extract the distribution files

-   For Windows:\
    Run the self-extracting mimic-windows.exe from an Administrator account.

-   For Linux:\
    `gunzip -c mimic-linux.tar.gz | tar xf -`

    The extracted files are:

    README - further release-specific instructions\
    license.txt - the license agreement\
    install - the installation script\
    mimic.tar - the product

> You are not done, please review the instructions in the next step.

Reviewer: ```suggestion

```

#### Step 3: Further instructions

> -   If you are **updating from an older release**

Reviewer: ```suggestion
-   If you are **updating from an older version**
```

>     -   make sure you **terminate MIMIC** (use `File->Terminate`) before running the install program.
> 
>     -   **Do not uninstall the older release** (this way you can always fallback in case of problems).
> 
>     -   Install this release in a **different directory from the older release**. MIMIC installs in these folders by default:

Reviewer: ```suggestion
    -  **Terminate MIMIC** (use `File->Terminate`) before running the install program.
    -  **Do not uninstall the older release** in the event a fallback is required
    -   Install this release in a **different directory from the older version**. MIMIC installs in these folders by default:
```
